### ISSUE TITLE
b'Segmentation fault (core dumped)'

### ISSUE THREAD

**Issue by horcsab** [eoyilmaz/displaycal-py3#359]
Hi,
I have a problem with Displaycal. When starting the measurement, I get this error message. What could be the problem?

```shell
source ./.venv/bin/activate; \
displaycal
Acquired lock file: <DisplayCAL.main.AppLock object at 0x7498497022a0>
displaycal 3.9.12 2024-05-02T15:07:23Z
ubuntu 24.04 noble x86_64
Python 3.12.3 (main, Apr 10 2024, 05:33:47) [GCC 13.2.0]
Faulthandler 
wxPython 4.2.1 gtk3 (phoenix) wxWidgets 3.2.2.1
Encoding: utf-8
File system encoding: utf-8
Loading /home/makka/.config/DisplayCAL/DisplayCAL.ini
listening
writing to lock file: port: 15411
Starting up...
SDL2: libSDL2-2.0.so.0
SDL2_mixer: libSDL2_mixer-2.0.so.0
Audio module: SDL 2.0
Enumerating display devices and communication ports...
/usr/bin
ArgyllCMS 3.1.0
Argyll has virtual display support
...ok.
Checking video card gamma table access for display 1...
Dispwin: Warning - new_dispwin: Expected VideoLUT depth 8 doesn't match actual 10
VideoLUT has 1024 entries, interpolating to 256
Dispwin: Warning - new_dispwin: Expected VideoLUT depth 8 doesn't match actual 10
Dispwin: Warning - new_dispwin: Expected VideoLUT depth 8 doesn't match actual 10Verify: 'test.cal' IS loaded (discrepancy 0.0%)
Dispwin: Warning - new_dispwin: Expected VideoLUT depth 8 doesn't match actual 10
...ok.
Initializing GUI...

input_value_clipping_bmp should have been hidden
output_profile_ctrl should have been hidden
...ok.
Ready.
Setting up scripting host at 127.0.0.1:15411
Check for application update...
DisplayCAL is up-to-date.
ArgyllCMS is up-to-date.
17:54:57: Debug: window wxComboBox@0x10dcafb0 ("whitepoint_colortemp_textctrl") lost focus even though it didn't have it
17:54:57: Debug: window wxComboBox@0x10dcafb0 ("whitepoint_colortemp_textctrl") lost focus even though it didn't have it
17:54:57: Debug: window wxComboBox@0x10dcafb0 ("whitepoint_colortemp_textctrl") lost focus even though it didn't have it
17:54:57: Debug: window wxComboBox@0x10dcafb0 ("whitepoint_colortemp_textctrl") lost focus even though it didn't have it
17:54:57: Debug: window wxComboBox@0x10dcafb0 ("whitepoint_colortemp_textctrl") lost focus even though it didn't have it
17:54:57: Debug: window wxComboBox@0x10dcafb0 ("whitepoint_colortemp_textctrl") lost focus even though it didn't have it
17:54:57: Debug: window wxComboBox@0x10dcafb0 ("whitepoint_colortemp_textctrl") lost focus even though it didn't have it
17:54:57: Debug: window wxComboBox@0x10dcafb0 ("whitepoint_colortemp_textctrl") lost focus even though it didn't have it
Reloading /home/makka/.config/DisplayCAL/DisplayCAL.ini
┌──────────────────────────────────────────────────────────────────────────────┐
│ b'Segmentation fault (core dumped)'                                          │
└──────────────────────────────────────────────────────────────────────────────┘
```

**Comment by eoyilmaz** [comment]
I don't think we are properly supporting Python 3.12 yet. This might be related to that. Also please try installing ArgyllCMS 3.2.0 that might help too.

**Comment by eoyilmaz** [comment]
Can you try the latest development branch, Python 3.12 should work properly now

**Comment by eoyilmaz** [comment]
`Python 3.12` and `wxPython 4.2.1` was not playing along nicely. This should be fixed now with `wxPython 4.2.2`
